boards/arm/stm32/stm32f4discovery/Kconfig

```diff
@@ -36,7 +36,7 @@ config STM32_ROMFS_IMAGEFILE
 
 config STM32F4DISCO_USBHOST_STACKSIZE
 	int "USB host waiter stack size"
-	default 1024
```

Comment:
It is inappropriate to increase the size of this tuned stack size.  Let's not do that.